src/routes/user.js

```diff
@@ -90,7 +90,7 @@ router.get('/profile/:id', validator(profileViewSchema), asyncWrapper(verifyToke
  * @swagger
  *
  * /profile/:userId/following:
- *   post:
+ *   patch:
```

Comment:
Why was this changed?

Reply:
I changed that part because the route is a patch route, not post route.

Reply:
I changed it because is a patch route and not a post route.